helix-core/src/main/java/org/apache/helix/model/InstanceConfig.java

```diff
@@ -408,6 +410,18 @@ public void setInstanceEnabledForPartition(String resourceName, String partition
     }
   }
 
+  /**
+   * Get maximum allowed running task count on this instance
+   * @return the maximum task count
+   */
+  public int getMaxConcurrentTask() {
```

Comment:
make this config option also available in cluster config? In most of cases, user would choose same value for all of its nodes, this makes them easier to set just once instead of in each instance config.

If the value is set in both cluster config and some of instance config, the value in instance config can take effect.